src/component/navbar/index.js

```diff
@@ -0,0 +1,30 @@
+import logo from "../../assets/Screenshot 2024-08-28 200018.png"
+import Popup from  "../Popup"
+import "./index.css";
+
+
+export default function Navbar() {
+
+  /*const onSearchChange = debounce((e) => {
+    if (e.target.value.length > 2) { 
+      dispatch(getMoviesBySearch(e.target.value));
+      dispatch(setcurrentPage(1));
+    }else{
+      dispatch(getMovies())
+    }
+  }, 500);*/
+
+  return (
+    <header className="navbar">
+      <img id="logo" src={logo} alt="logo"/>
+      <div className="Seachbar">
+        <svg viewBox="0 0 24 24" aria-hidden="true" className="icon" alt="searchlogo">
+          <g><path d="M21.53 20.47l-3.66-3.66C19.195 15.24 20 13.214 20 11c0-4.97-4.03-9-9-9s-9 4.03-9 9 4.03 9 9 9c2.215 0 4.24-.804 5.808-2.13l3.66 3.66c.147.146.34.22.53.22s.385-.073.53-.22c.295-.293.295-.767.002-1.06zM3.5 11c0-4.135 3.365-7.5 7.5-7.5s7.5 3.365 7.5 7.5-3.365 7.5-7.5 7.5-7.5-3.365-7.5-7.5z"></path></g>
+        </svg>
+        <input type="text" maxLength="40"  placeholder="Search the movies" />
+      </div>
```

Comment:
can we create a seperate component for search bar and call it here , Like 
`<SearchBox onSubmit={handleSearchCallback} />`